WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -16,7 +16,7 @@ final class PaymentCaptureOrchestrator {
     private var walletSuppressionRequestToken: PKSuppressionRequestToken?
 
     func collectPayment(for order: Order,
-                        statementDescriptor: String?,
+                        paymentGatewayAccount: PaymentGatewayAccount,
```

Comment:
This is the relevant part of the solution. Because at this point we have a reference to the gateway, we pass it down